go.sum

```diff
@@ -1,3 +1,4 @@
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```

Comment:
I notice this project is unmaintained.

Reply:
thats an indirect dependency introduced by `github.com/urfave/cli` which as far as i see is maintained so if any issues arise, i guess they will either remove this dependency/fork/use something else

Reply:
@martinkennelly how did you check the project is unmaintained, is it via some tools? just curious since I don't know how this could be done.

Reply:
@adrianchiris spot on - its not a problem for this patch.
@zshi-redhat No tool unfortunately. I manually went through the added repositories to check.